packages/docs/docs.json

```diff
@@ -1047,7 +1047,7 @@
             "resolved": "boolean",
             "references": {}
           },
-          "mutable": true,
+          "mutable": false,
```

Comment:
Wasn't expecting to see this here. I've just pulled down v3 develop and I'm still seeing the mutable as true.

Reply:
Just checked and the prop itself doesn't have mutable `@Prop() visible: boolean = true;` so I think it must have changed at some point and the docs never got updated

Reply:
yeah, looks like it was changed a couple of days ago: https://github.com/mi6/ic-ui-kit/commit/f9ae560d91688a60e7a5a261bbd599ad2b1c82f6#diff-1bec309673391985a7039927575aa48e135026b67f8a2141c97fd974edde7e21